app/components/agenda/agenda-header/agenda-version-actions.hbs

```diff
@@ -7,12 +7,12 @@
   class="auk-u-hidden@print"
 >
   {{#if @currentAgenda.status.isDesignAgenda}}
-  <AuButton data-test-agenda-version-actions-approve-agenda @skin="link" {{on "click" this.openConfirmApproveAgenda}}
+  <AuButton data-test-agenda-version-actions-approve-agenda @skin="link" {{on "click" this.openConfirmApproveAgendaSidebar}}
```

Comment:
The naming of `openConfirmApproveAgendaSidebar` and `showConfirmForApprovingAgendaSidebar` is a bit confusing here. Why "sidebar"? 

The property controls whether or not the `AgendaCheck` component is shown, so `openAgendaCheck` and `showAgendaCheck` would be more logical, no?